### ISSUE TITLE
How to modify the Timeseries forecasting for weather prediction example to increase the number of predictions?

### ISSUE THREAD

**Issue by skanskan** [keras-team/keras-io#1205]
Hello.

I've been following the "Timeseries forecasting for weather prediction" code found here:
https://keras.io/examples/timeseries/timeseries_weather_forecasting/

The article says 
"The trained model above is now able to make predictions for 5 sets of values from validation set."

And it uses this code to get predictons and plot them:

```
def show_plot(plot_data, delta, title):
    labels = ["History", "True Future", "Model Prediction"]
    marker = [".-", "rx", "go"]
    time_steps = list(range(-(plot_data[0].shape[0]), 0))
    if delta:
        future = delta
    else:
        future = 0

    plt.title(title)
    for i, val in enumerate(plot_data):
        if i:
            plt.plot(future, plot_data[i], marker[i], markersize=10, label=labels[i])
        else:
            plt.plot(time_steps, plot_data[i].flatten(), marker[i], label=labels[i])
    plt.legend()
    plt.xlim([time_steps[0], (future + 5) * 2])
    plt.xlabel("Time-Step")
    plt.show()
    return


for x, y in dataset_val.take(5):
    show_plot(
        [x[0][:, 1].numpy(), y[0].numpy(), model.predict(x)[0]],
        12,
        "Single Step Prediction",
    )
```

What do I need to modify if I want to get predictions for the next 500 intervals instead of just 5?
`future + 500`
and 
`dataset_val.take(500)`

Or something else?

In my computer instead of using "sampling_rate=6" I have modified the frequency of data from the beginning, and kept only 1 hour information and I'm using "sampling_rate=1"

The configuration at the beginning also says:
```
split_fraction = 0.715
train_split = int(split_fraction * int(df.shape[0]))
step = 6

past = 720  
future = 72
learning_rate = 0.001
batch_size = 256
epochs = 10

```

What values do I need to use for past and future?
future = 500.
past = ?

**Comment by dhantule** [comment]
Hi @skanskan, thanks for reporting this. 

To predict for next 500 intervals, you can keep the future and past values the same i.e 
`past = 720 , future = 72`, 

however you'll have to modify the for loop to take samples from validation dataset 
`for i, (x, y) in enumerate(dataset_val.take(500)):`
